src/Groups/GroupDetails/GroupDetailsContainer.js

```diff
@@ -17,7 +17,7 @@ import {
 } from '../../common/const';
 import {
   groupByUrlIdResource,
-  fiscalYearsResource,
+  fiscalYearsResource, fundsResource,
```

Comment:
do we have some kind of tax for taking extra lines?

Reply:
Updated